builtin_string.go

```diff
@@ -371,6 +371,39 @@ func (r *Runtime) stringproto_match(call FunctionCall) Value {
 	panic(r.NewTypeError("RegExp matcher is not a function"))
 }
 
+func (r *Runtime) stringproto_matchAll(call FunctionCall) Value {
+	r.checkObjectCoercible(call.This)
+	regexp := call.Argument(0)
+	if regexp != _undefined && regexp != _null {
+		if isRegexp(regexp) {
+			if o, ok := regexp.(*Object); ok {
+				flags := o.Get("flags")
+				r.checkObjectCoercible(flags)
+				if !strings.Contains(flags.String(), "g") {
```

Comment:
```suggestion
				if !strings.Contains(flags.toString().String(), "g") {
```
The difference is that the `String()` on a `Symbol` will return its description, whereas `toString()` will throw a TypeError as specified by the `ToString` operation.